### DISCUSSION TITLE
Compatibility with netbox 3.6.x

### DISCUSSION THREAD

**Discussion by rPr0x** [Onemind-Services-LLC/netbox-secrets#96] [General]
Forecast to release the version compatible with Netbox 3.6.0?

I'm installing via docker and it's returning this error:

2,650 django.core.exceptions.ImproperlyConfigured: Plugin netbox_secrets requires NetBox maximum version 3.5.99.
------
failed to solve: process "/bin/sh -c SECRET_KEY=\"dummydummydummydummydummydummydummydummydummydummy\" /opt/netbox/venv/bin/python /opt/netbox/netbox/manage.py collectstatic --no-input" did not complete successfully : exit code: 1

**Comment by abhi1693** [comment]
We will try to get this closed this week: https://github.com/Onemind-Services-LLC/netbox-secrets/issues/95
